crates/gateway-types/src/reply.rs

```diff
@@ -474,6 +479,10 @@ pub mod transaction {
         pub transaction_hash: StarknetTransactionHash,
     }
 
+    pathfinder_common::version_check!(
+        Mainnet < 0 - 11 - 0,
+        "Remove sender_address alias - beware storage impact"
+    );
```

Comment:
Probably not; but at the time I was hoping to prompt us to revisit decoupling storage vs gateway types. For example, we could check if historic blocks still contain this or alias or not. In which case it might only be our own storage "holding us back".